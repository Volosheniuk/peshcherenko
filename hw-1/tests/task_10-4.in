q 1234 1.9999 1.999999999999
