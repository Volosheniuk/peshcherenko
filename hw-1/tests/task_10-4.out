q
1234
1.999900
2.000000
